rpm/python-mdnsbridge.spec

```diff
@@ -5,6 +5,8 @@ License: 		Internal Licence
 Summary: 		mDNS to HTTP bridge service
 
 Source0: 		%{name}-%{version}.tar.gz
+Source1:		ips-api-mdnsbridge.conf
+Source2:		python-mdnsbridge.service
```

Comment:
How do these get copied into `build/rpm/SOURCES`? I can see a Makefile target for the source tarball, but not these config files.

Reply:
Is this not covered by the lines under %files? I'm not totally familiar with these so it may not be.

Reply:
That refers to their location in the target filesystem - the `%files` list is a list of files owned by this package